src/tiledb/cloud/file/ingestion.py

```diff
@@ -82,7 +39,7 @@ def add_arrays_to_group_udf(
             if tiledb.object_type(group_uri) == "group":
                 with tiledb.Group(group_uri, mode="w") as group:
                     logger.debug(
-                        f"Adding to Group {group_uri} the following: {array_uris}"
+                        "Adding to Group %s the following: %s" % (group_uri, array_uris)
```

Comment:
Like `logger.debug("Adding to group %s the following: %s", group_uri, array_uris)` (i.e., the %-formatting gets done inside `logger.debug`, so it doesn’t have to do anything at all if the log message will never be emitted)